lib/marqeta/api_object.rb

```diff
@@ -32,7 +35,7 @@ def self.object_list(klass, endpoint)
       while is_more
         paginated_params = {
           count: QUERY_RESULTS_COUNT,
-          start_index: start_index
+          start_index:
```

Comment:
Can you have a hash key without a value like this?

Reply:
Ah cool I hadn't seen this https://blog.saeloun.com/2021/09/28/ruby-allow-value-omission-in-hash-literals/